SensioLabs/Security/Command/SecurityCheckerCommand.php

```diff
@@ -88,64 +84,23 @@ protected function execute(InputInterface $input, OutputInterface $output)
             return 1;
         }
 
-         if ('json' === $input->getOption('format')) {
-             $output->write(json_encode($vulnerabilities, JSON_PRETTY_PRINT));
-         } else {
-             $this->displayResults($output, $input->getArgument('lock'), $vulnerabilities);
-         }
-
-        if ($this->checker->getLastVulnerabilityCount() > 0) {
-            return 1;
+        switch ($input->getOption('format')) {
```

Comment:
this might be good in a getFormatter method, where we just return the instance, instead of setting it, then breaking.